ynr/apps/api/templates/api/api-home.html

```diff
@@ -13,6 +13,20 @@
 * Please give credit to Democracy Club when reusing this data.
 * Please [read the full terms]({% url "api-terms" %}).
 
+### Authentication
+
+The API has a rate limit of 10 requests per minute for unauthenticated users.
+{% if request.user.is_authenticated %}
+To authenticate, grab the token from <a href="{% url "my_profile" %}">your profile page</a>.
+
+Pass this token in with `auth_token={{ request.user.auth_token.key }}`.
```

Comment:
Do we want to render the token here if we are telling them to grab it from the profile page? Not a big deal but maybe we do one or the other.